test/cases/5.7_math.yaml

```diff
@@ -0,0 +1,27 @@
+tests:
+  - desc: '5.7 Math'
+  - desc: '5.7.5 ln() : Decimal'
+  - desc: '** Can take the natural logarithm of the number'
+    expression: Math.n1.ln()
+    result: [0]
+  - desc: '** Empty result when taking logarithm from empty collection'
+    expression: Math.n2.ln()
+    result: []
+  - desc: '** Error taking logarithm due to too many input parameters'
+    expression: Math.n3.ln(n4)
+    error: true
+  
+  - desc: '5.7.6  log(base : Decimal) : Decimal'
```

Comment:
Needs a test for a missing base.  From http://hl7.org/fhirpath/2019May/index.html#logbase-decimal-decimal:
"If base is empty, the result is empty."

Reply:
Okay, the new test looks good, but although it does return empty, it also complains about the arity being wrong:
```
$ bin/fhirpath -e '(100).log()' -r {}
### log wrong arity: got 0
fhirpath((100).log()) =>
[]
```
Take a look at how other functions handle optional arguments.